Dangerfile.swift

```diff
@@ -3,17 +3,17 @@ import Danger
 
 let danger = Danger()
 
-// if danger.github.pullRequest.branch_for_base != "develop" {
+// if danger.git.branch_for_base != "develop" {
 //     fail("Please re-submit this PR to develop, we may have already fixed your issue.")
 // }
 
 // Pull request size
-let bigPRThreshold = 200
+let bigPRThreshold = 600
 let additions = danger.github.pullRequest.additions!
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/dbgarcia/danger-bot/blob/4ad3f65193926f1211a9cfb888b46cbbba684b48/Dangerfile.swift#L12"></span>Force unwrapping should be avoided.<br />
<code>force_unwrapping</code> <code>Dangerfile.swift:12</code></del></td>
    </tr>
  </tbody>
</table>